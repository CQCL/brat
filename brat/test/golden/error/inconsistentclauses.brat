f(Bool, Nat) -> Nat
f(true, n) = 1
f(b) = 2
