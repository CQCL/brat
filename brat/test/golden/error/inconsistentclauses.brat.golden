Error in test/golden/error/inconsistentclauses.brat@1:1-3:9
f(Bool, Nat) -> Nat
^^^^^^^^^^^^^^^^^^^
f(true, n) = 1
^^^^^^^^^^^^^^
f(b) = 2
^^^^^^^^

  Unification error: Pattern doesn't match expected length for constructor args

